pinpointsmscallbacks/README.md

```diff
@@ -0,0 +1,33 @@
+# Purpose
+
+This lambda gets the Pinpoint receipt from AWS and then calls a task to update the notification in Notify's DB.
+
+## Lambda Invocation
+
+1. SMS gets sent through Notify
+1. The SMS receipt written to 2 different cloudwatch log-groups: PinpointDirectPublishToPhoneNumber and PinpointDirectPublishToPhoneNumber/Failure
+1. The pinpoint_to_sqs_sms_callbacks lambda is a subscription-filter on the above two log-groups.
+1. The lambda takes the sms receipt and sends it to a celery queue `eks-notification-canada-cadelivery-receipts` with a task `process-pinpoint-result`
+
+
+## How it works
+![PlantUML model](./sms-callback.png)
```

Comment:
We might want a flow proper to this new lambda. I think here it will target an inexistent file if I am not mistaken.

Reply:
ah, right, I meant to mermaid up a diagram here - added now.